pkg/rhcos/baseuri.go

```diff
@@ -0,0 +1,23 @@
+package rhcos
+
+import (
+	"context"
+	"net/url"
+
+	"github.com/pkg/errors"
+)
+
+// BaseURI fetches the BaseURI where images can be downloaded from
+func BaseURI() (string, error) {
```

Comment:
This information is rather kept private to the package... Please use or add other image functions that provide th boot-image directly.

Reply:
Currently the downloader container we're using requires the base URI

https://github.com/openshift/ironic-rhcos-downloader/blob/master/get-resource.sh

We can rework that, or pass the full URL to the openstack image perhaps.  cc @derekhiggins

Reply:
> We can rework that, or pass the full URL to the openstack image perhaps. cc @derekhiggins 

Sounds fine to me, let me know if you want to update the parameters the downloader container expects,